### DISCUSSION TITLE
add hard limit max online per permission

### DISCUSSION THREAD

**Discussion by mani1232** [Zrips/CMI#6841] [Enhancement]
for example: default 0/50 permission vip can join if online <=60, permission vip+ can join if online <=65

**Comment by CloudeLecaw** [comment]
vip slots for join when the server is shown aas full to not vip players?

well sadly its not up to me to desice this, but personaly i hope this never will be added^^

**Reply by mani1232** [reply]
It can display at least 9999 for not vip players, but not vip players can only enter up to their max value, for example, 50
